llvm/test/CodeGen/PowerPC/crsave.ll

```diff
@@ -15,11 +15,12 @@ entry:
 }
 
 ; PPC32-LABEL: test_cr2:
-; PPC32: stwu 1, -32(1)
-; PPC32: stw 31, 28(1)
+; PPC32: stwu 1, -[[#%u,AMT:]](1)
+; PPC32: stw 31, [[#AMT - 4]](1)
```

Comment:
would you please rebase the case after change https://github.com/llvm/llvm-project/commit/3196005f6bedbed61a86626a9e4f8fee7437a914 ? Thanks.

Reply:
Done. I stopped adding CFI to test_cr234, because cloberAllNvCrField has similar CFI.